openwisp_notifications/templates/openwisp_notifications/emails/notification.html

```diff
@@ -70,60 +104,19 @@
     background-color: #DCFAE6;
   }
 
-  a:link:hover .alert,
-  a:link:focus .alert {
-    background-color: #fff9c466
-  }
-
-  .alert .title {
-    margin-left: 0.5rem;
-    padding: 0.13rem 0;
-  }
-
-  .alert .title p {
-    line-height: 1.25rem;
-    font-size: 0.875rem;
-    display: inline-block;
-    vertical-align: middle;
-    margin: 0 auto;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    color: #0F1729;
-    max-width: 550px;
-    width: 100%;
-    white-space: nowrap;
-  }
-
-  .alert .title {
-    display: inline-block;
-    width: 80%;
-  }
-
-  .right-arrow-container {
-    width: 6%;
+  .alert hr {
```

Comment:
What about:
```suggestion
  .alert .hr {
```

Related to https://github.com/openwisp/openwisp-utils/pull/493#pullrequestreview-3108055490.